src/solver/optim-model-filler/ComponentFiller.cpp

```diff
@@ -421,21 +478,23 @@ void ComponentFiller::addObjectives(const LinearProblemApi::FillContext& ctx)
           objective.expression().RootNode());
 
         auto& pb = optimEntityContainer_.Problem();
```

Comment:
```c++
auto& pb = optimEntityContainer_.Problem();
```
A reference to the linear problem is instantiated inside the loop on objectives. 
So we're doing the same instantiation  several times. 
It should be done before this loop.